Make.config

```diff
@@ -219,7 +219,9 @@ SIMULATORWATCH_OBJC_CFLAGS = $(SIMULATORWATCH_CFLAGS) $(COMMON_SIMULATOR_OBJC_CF
 
 DEVICEWATCH_SDK         = $(XCODE_DEVELOPER_ROOT)/Platforms/WatchOS.platform/Developer/SDKs/WatchOS$(WATCH_SDK_VERSION).sdk
 DEVICEWATCH_CFLAGS      = -arch armv7k -mwatchos-version-min=$(MIN_WATCHOS_SDK_VERSION) -isysroot $(DEVICEWATCH_SDK) $(CFLAGS) -fembed-bitcode $(IOS_COMMON_DEFINES)
+DEVICEWATCH64_32_CFLAGS = -arch arm64_32 -mwatchos-version-min=$(MIN_WATCHOS_SDK_VERSION) -isysroot $(DEVICEWATCH_SDK) $(CFLAGS) -fembed-bitcode $(IOS_COMMON_DEFINES)
```

Comment:
arm64_32 watch needs a different min version. Here we pass `$(MIN_WATCHOS_SDK_VERSION)` which is 2.0, and I think the tooling will "force upgrade" it to 5.0. Afaik the first public arm64_32 watchOS was 5.1, that's why we choose that value in the mono SDK: https://github.com/mono/mono/blob/5232408f7ad6edd95429769970d314c741e442e1/sdks/versions.mk#L43

Note: This is merely needed to avoid warnings like those:
```
warning MT5203 : Native linking warning : warning: object file (/Library/Frameworks/Xamarin.iOS.framework/Versions/12.6.0.59/SDKs/Xamarin.WatchOS.sdk/usr/lib/libmonosgen-2.0.a(libmonoutils_la-mono-time.o)) was built for newer watchOS version (5.1) than being linked (5.0)
```